splicemachine/features/feature_store.py

```diff
@@ -1138,14 +1139,16 @@ def alter_pipe(self, name: str, func: Optional[Union[str, Callable]] = None, des
         Alters an existing version of a pipe. Use this method when you want to make changes to a version of a pipe
         that has no dependencies, or when you want to change version-independent metadata, such as description.
 
-        :param name: The training set name.
+        :param name: The pipe name.
         :param func: The actual python function or sql query used in the transformation
         :param description: (Optional[str]) An optional description of the pipe
         :param version: The version to alter (either an int or 'latest'). Default 'latest'
         :return:
         """
         assert name != "None", "Name of pipe cannot be None!"
         assert func or description, "Please enter either a function or description"
+        if isinstance(version, str) and version != 'latest':
```

Comment:
do we need to check that the pipe is > 0? Or will that not be a problem on the server?